packages/airnode-node/src/evm/networks.ts

```diff
@@ -1,14 +1,18 @@
 import { ethers } from 'ethers';
+// Documentation link: https://docs.api3.org/airnode/v0.2/reference/airnode-addresses.html
 
 /*
  * Networks:
  *
  * 1 - Ethereum Mainnet
  * 3 - Ethereum Ropsten
  * 4 - Ethereum Rinkeby
+ * 5 - Ethereum goerli
```

Comment:
Just for consistency with the other networks
```suggestion
 * 5 - Ethereum Goerli
```